DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -323,8 +324,8 @@ public void RegisterCommands(Type type, Action<ApplicationCommandsPermissionCont
         /// </summary>
         /// <param name="client">The client.</param>
         /// <param name="e">The ready event args.</param>
-        internal async Task UpdateAsync(DiscordClient client, ReadyEventArgs e)
-            => await this.UpdateAsync();
+        internal Task UpdateAsync(DiscordClient client, ReadyEventArgs e)
+            => _ = Task.Run(async () => await this.UpdateAsync());
```

Comment:
Why this change?

Reply:
Its unrelated to T109 really but noticed the UpdateAsync had been thrown warnings from gateway timeouts which this should get rid of.

It can be taken out if wanted as the warnings currently happen on latest nightly too

Reply:
Interesting. I'd have to investigate further on that. It just appears this is doing the same thing in a fire/forget manner